custom/icds_reports/ucr/reports/ls/ls_v2_cbe.json

```diff
@@ -0,0 +1,164 @@
+        "required": false,
+        "slug": "awc_id",
+        "field": "awc_id",
+        "choice_provider": {
+          "type": "location"
+        },
+        "ancestor_expression": {
+          "field": "supervisor_id",
+          "location_type": "supervisor"
+        },
+        "display": "Filter by AWW"
+      },
+      {
+        "compare_as_string": false,
+        "show_all": true,
+        "datatype": "string",
+        "type": "dynamic_choice_list",
+        "required": false,
+        "slug": "supervisor_id",
+        "field": "supervisor_id",
+        "choice_provider": {
+          "type": "location"
+        },
+        "display": "Filter by Supervisor"
+      }
+    ],
+    "columns": [
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "transform": {
+          "type": "custom",
+          "custom_type": "owner_display"
+        },
+        "column_id": "awc_id",
+        "field": "awc_id",
+        "calculate_total": false,
+        "type": "field",
+        "display": "awc_id",
+        "aggregation": "simple"
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "simple",
+        "column_id": "date_cbe_organise",
+        "field": "date_cbe_organise",
+        "transform": {},
+        "calculate_total": false,
+        "type": "field",
+        "display": "date_cbe_organise"
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "simple",
+        "column_id": "theme_cbe",
+        "field": "theme_cbe",
+        "transform": {},
+        "calculate_total": false,
+        "type": "field",
+        "display": "theme_cbe"
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "simple",
+        "column_id": "count_targeted_beneficiaries",
+        "field": "count_targeted_beneficiaries",
+        "transform": {},
+        "calculate_total": false,
+        "type": "field",
+        "display": "count_targeted_beneficiaries"
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "simple",
+        "column_id": "count_other_beneficiaries",
+        "field": "count_other_beneficiaries",
+        "transform": {},
+        "calculate_total": false,
+        "type": "field",
+        "display": "count_other_beneficiaries"
+      },
+      {
+        "sortable": false,
+        "description": null,
+        "format": "default",
+        "aggregation": "simple",
+        "column_id": "submitted_on",
+        "field": "submitted_on",
+        "transform": {},
+        "calculate_total": false,
+        "type": "field",
+        "display": "submitted_on"
+      }
+    ],
+    "sort_expression": [
+      {
+        "field": "date_cbe_organise",
```

Comment:
If there isn't yet a `supervisor_id, awc_id, date_cbe_organise` on the data source, that would improve the performance of this report.

Reply:
Will update. The only index on the data source is on awc_id: https://github.com/dimagi/commcare-hq/blob/fc5fd8c590eb4f9703449934269a69efe6ed69d6/custom/icds_reports/ucr/data_sources/cbe_form.json#L74

Reply:
https://github.com/dimagi/commcare-hq/pull/27225